app/assets/javascripts/modules/external_users/claims/FeeCalculator.js

```diff
@@ -93,7 +118,7 @@ moj.Modules.FeeCalculator = {
   },
 
   displayHelp: function(context, show) {
-    var $help = $(context).closest('.fixed-fee-group').find('.help-wrapper.form-group');
+    var $help = $(context).siblings('.help-wrapper.form-group');
```

Comment:
not sure how safe `siblings()` is if things move about in future? @cleargif could clarify. I know `closest()` searches up the dom and then find searches down